packages/api/src/tasks/checkAlerts.ts

```diff
@@ -58,25 +56,110 @@ export const buildLogSearchLink = ({
   return url.toString();
 };
 
-const buildEventSlackMessage = ({
+// TODO: should link to the chart instead
+export const buildChartLink = ({
+  dashboardId,
+  endTime,
+  granularity,
+  startTime,
+}: {
+  dashboardId: string;
+  endTime: Date;
+  granularity: string;
+  startTime: Date;
+}) => {
+  const url = new URL(`${config.FRONTEND_URL}/dashboards/${dashboardId}`);
+  const queryParams = new URLSearchParams({
+    from: startTime.getTime().toString(),
+    granularity,
+    to: endTime.getTime().toString(),
+  });
+  url.search = queryParams.toString();
+  return url.toString();
+};
+
+const buildChartEventSlackMessage = ({
+  alert,
+  dashboard,
+  endTime,
+  granularity,
+  group,
+  startTime,
+  totalCount,
+}: {
+  alert: AlertDocument;
+  endTime: Date;
+  dashboard: {
+    id: string;
+    name: string;
+    chart: {
+      id: string;
+      name: string;
+      series: IDashboard['charts'][0]['series'][0];
+    };
+  };
+  granularity: string;
+  group?: string;
+  startTime: Date;
+  totalCount: number;
+}) => {
+  const mrkdwn = [
+    `*<${buildChartLink({
+      dashboardId: dashboard.id,
+      endTime,
+      granularity,
+      startTime,
+    })} | Alert for "${dashboard.chart.name}" in "${dashboard.name}">*`,
+    ...(group != null ? [`Group: "${group}"`] : []),
+    `${totalCount} lines found, expected ${
+      alert.type === 'presence' ? 'less than' : 'greater than'
+    } ${alert.threshold} lines`,
```

Comment:
```suggestion
    } ${alert.threshold}`,
```
![image](https://github.com/hyperdxio/hyperdx/assets/2781687/0ae0f969-84bb-4669-a08f-0ae708ed0daf)